src/main/resources/init_db.sql

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE IF NOT EXISTS books (
+    id BIGINT NOT NULL AUTO_INCREMENT PRIMARY KEY,
```

Comment:
```suggestion
    id BIGINT AUTO_INCREMENT PRIMARY KEY,
```
Primary key already provided not null constraint

Reply:
++. thank you, changed